sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLEnv.scala

```diff
@@ -35,7 +35,7 @@ private[hive] object SparkSQLEnv extends Logging {
   def init() {
     if (sqlContext == null) {
       val sparkConf = new SparkConf(loadDefaults = true)
-      // If user doesn't specify the appName, we want to get [SparkSQL::localHostName] instead of
+      // If user doesn't specify the appName, we will get [SparkSQL::IP] instead of
```

Comment:
What is SparkSQL::IP?
I think this is not worth a JIRA / PR relative to the time spent looking at it
What problem does this cause a user?